data/governments.txt

```diff
@@ -937,6 +954,21 @@ government "Lunarium (Hidden)"
 		"Heliarch" -.01
 		"Pirate" -.01
 
+government "Magic Asteroid"
+	"display name" "???"
+	swizzle 0
+	language "???"
+	
+	"attitude toward"
+		"Ka'sei (12A)" -.01
+		"Uninhabited" -.01
+
+	"player reputation" 10
+	"friendly hail" "friendly ka'het"
```

Comment:
They hail? And if so, they use the Ka’het hails?

Reply:
Removed